src/Microsoft.TestPlatform.Client/Resources/Resources.resx

```diff
@@ -120,6 +120,9 @@
   <data name="InvalidStateForExecution" xml:space="preserve">
     <value>The test run could not be executed because the initial state was invalid.</value>
   </data>
+  <data name="TestSessionTimeoutMessage" xml:space="preserve">
+    <value>Canceling test run as it crossed testSessionTimeout: {0} milliseconds.</value>
```

Comment:
`testSessionTimeout` looks like an internal implementation name, it may not be exposed in user message?